apps/web-app/app/routes/migrate.tsx

```
@@ -0,0 +1,317 @@
+                    onChange={(e) => setTargetStorage(e.target.value as 'local')}
+                  />
+                  Migrate from Cloud to Local Storage
+                  <p className="option-description">
+                    Offline access and faster performance
+                  </p>
+                </label>
+              </div>
+            </div>
+
+            <div className="wizard-actions">
+              <button onClick={() => setStep('configure')} className="button primary">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)